tests/basic_correctness/test_chunked_prefill.py

```diff
@@ -78,10 +66,10 @@ def test_models(
     )
 
 
-@pytest.mark.parametrize("kv_cache_dtype,model",
-                         [("fp8_e5m2", m)
-                          for m in E5M2_KV_MODELS] + [("fp8_e4m3", m)
-                                                      for m in E4M3_KV_MODELS])
+@pytest.mark.parametrize(
+    "kv_cache_dtype,model",
+    [("fp8_e4m3",
+      "nm-testing/TinyLlama-1.1B-compressed-tensors-kv-cache-scheme")])
```

Comment:
Can you keep a test for fp8_e5m2?

Reply:
I covered e5m2 in `test_fp8.py` already. I could add this back if you think it's critical to cover e5m2 for chunked prefill. Plz let me know.

Reply:
Okay that should be fine without then